automated install/basic-install.sh

```diff
@@ -1754,8 +1754,25 @@ FTLinstall() {
   # Move into the temp ftl directory
   pushd "$(mktemp -d)" > /dev/null || { echo "Unable to make temporary directory for FTL binary download"; return 1; }
 
-  # Always replace pihole-FTL.service
-  install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/pihole-FTL.service" "/etc/init.d/pihole-FTL"
+  # Determine if systemd is used on this system
+  if file "$(which init)" | grep "systemd" &> /dev/null; then
+    # Use systemd unit
+    # Always replace pihole-FTL.service (systemd unit)
+    install -T -m 0644 "${PI_HOLE_LOCAL_REPO}/advanced/pihole-FTL.systemd" "/etc/systemd/system/pihole-FTL.service"
+    install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/pihole-FTL-prestart.sh" "/opt/pihole/pihole-FTL-prestart.sh"
+
+    # Remove old init.d script if present as it cannot coexist with the systemd unit we are installing here
+    if [ -e "/etc/init.d/pihole-FTL" ]; then
+      rm "/etc/init.d/pihole-FTL"
```

Comment:
This will leave dangling /etc/rc?.d/pihole-FTL symlinks.
Those should be cleaned up using ```update-rc.d pihole-FTL remove```